src/pseudopeople/utilities.py

```diff
@@ -166,6 +166,17 @@ def get_state_abbreviation(state: str) -> str:
         raise ValueError(f"Unexpected state input: '{state}'") from None
 
 
+def cleanse_integer_columns(column: pd.Series) -> pd.Series:
```

Comment:
I think it's better to set the values directly into the input `column` rather than concatting these two series and then sorting them. Something like this seems better:

```
column = column.copy()
float_mask = ~column.isna() & (column.str[-2] == ".")
column.loc[float_mask] = column.loc[float_mask].str[:-2]
return column
```

Reply:
Oh interesting. I don't love that you are assuming that the decimal is always at [-2]. Is that guaranteed? What about something like 
```
float_mask = ~column.isna() & (column.str.contains(".", regex=False))
column.loc[float_mask] = column.loc[float_mask].str.split(".").str[0]
```

Reply:
Ok I think I have this working now. The original concat (which I agree is pretty messy) is that all non-null values do need to be ocnverted to strings.